src/api/client.py

```
@@ -437,10 +571,34 @@ async def get_entry(self, entry_id: int) -> PasswordEntry:
+            # Need to encrypt using vault
+            from crypto.vault import get_vault
+            vault = get_vault()
+            if not vault.is_unlocked():
+                # Try to unlock with master password and salt
+                if hasattr(self, '_master_password') and hasattr(self.user_session, 'vault_salt'):
+                    vault.unlock(self._master_password, self.user_session.vault_salt)
+                else:
+                    raise ValueError("Vault is locked and cannot encrypt data")
+            
+            # Encrypt the entry data
+            encrypted_data = vault.encrypt_entry(entry_data)
```

Ruff (0.8.2)

Local variable `encrypted_data` is assigned to but never used

Remove assignment to unused variable `encrypted_data`

(F841)